frontend/components/CourseActions/components/CourseDescriptionAndStats.js

```diff
@@ -32,7 +59,7 @@ class CourseDescriptionAndStats extends React.Component {
 
   renderStat = (icon, stat) =>
     <li>
-      <div className="stat">{stat}</div>
+      <div className="stat category_and_author">{stat}</div>
```

Comment:
Hm why do all of these stats have the `category_and_author` class now?

<img width="213" alt="image" src="https://user-images.githubusercontent.com/7578559/154469067-0b95e175-8fc5-4d2f-bff7-13dae12d6b50.png">

Reply:
No idea lol, removed!